popup.html

```diff
@@ -86,6 +86,12 @@
 		</div>
 		<div class="error" id="error"></div>
 		<div class="content" id="content"></div>
+		<div class="labelPage" id="labelPage" style="display: none;">
+			<div id="checkboxes">
+				<label>Select labels</label>
```

Comment:
This should not be `<label>`, but rather something like a headline (e.g. `<h2>`) or even a `<fieldset>`, as documented on MDN: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input/checkbox#handling_multiple_checkboxes